examples/step-20/step-20.cc

```diff
@@ -14,7 +14,8 @@
  * ---------------------------------------------------------------------
 
  *
- * Author: Wolfgang Bangerth, Texas A&M University, 2005, 2006
+ * Authors: Wolfgang Bangerth, Texas A&M University, 2005, 2006;
+ *          (port to LinearOperator:) Matthias Maier, 2019
```

Comment:
Just remove both lines -- by now probably essentially everyone has doctored around this program at least once.